demo/full/styles/style.css

```diff
@@ -1,40 +1,66 @@
-body {
-  margin: 0px;
-}
-
 @font-face {
   font-family: "icons";
   src: url("../fonts/FontAwesome.otf");
   font-weight: normal;
   font-style: normal;
 }
 
+@font-face {
+  font-family: "Roboto" "Open Sans", sans-serif;;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Unexpected extra semicolon ([css:S1116](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS1116))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=312&issues=AWXYrlGz_eEZPtUmEleE&open=AWXYrlGz_eEZPtUmEleE)

[ISSUE_KEY]: AWXYrlGz_eEZPtUmEleE